config/config.go

```diff
@@ -231,8 +231,11 @@ func (c DatabaseConfig) GetHostname() string {
 
 type PostgreSQLConfig struct {
 	DatabaseConfig
-	SSL     bool   `json:"ssl,omitempty"`
-	SSLMode string `json:"sslMode,omitempty"`
+	SSL              bool                    `json:"ssl,omitempty"`
+	SSLMode          string                  `json:"sslMode,omitempty"`
+        CaCertSecret     apiv1.SecretKeySelector `json:"caCertSecret,omitempty"`
+        ServerCertSecret apiv1.SecretKeySelector `json:"serverCertSecret,omitempty"`
+        ServerKeySecret  apiv1.SecretKeySelector `json:"serverKeySecret,omitempty"`
```

Comment:
can you fix the lint here please https://github.com/argoproj/argo-workflows/actions/runs/3832454101/jobs/6522907398

Reply:
@rohankmr414  tried in visual studio and not getting any lint error but again I will cross check on lint errors.

Reply:
Can you run `make lint` in the repo directory? That'll fix it